libs/game---light/compat.ts

```diff
@@ -28,6 +28,18 @@ namespace game {
         return _scene
     }
 
+    export function pushScene() {
```

Comment:
`pushScene` and `popScene` here might be confusing to some. Are they needed? Kojac calls `push/popEventContext` directly.

Reply:
well, I wanted to have something cross-compatible between game and game---light, but maybe that should not be the goal?

Reply:
I lean toward keeping it out until there's a clear need. Are there any other missing APIs, or is this the only one?

Reply:
For my use case it was just this one.

I think we should remove game.pushEventContext from game---light and replace it with game.pushScene. This way whatever works in game---light should work the same in standard game, allowing for UI library targeting both.

I don't think there's much risk of confusion here - the Scene class in game---light has almost no fields, and things like Sprites are missing altogether.

Though maybe the UI library could do its own drawing handlers and just use control.*EventContext...

Reply:
That makes sense.